src/main/java/org/example/main.java

```diff
@@ -41,12 +41,13 @@ public static void main(String[] args) {
 
           /**  add some information to this map - any detail you consider relevant
             you will use a HashMap with the following structure <Person, List<Hobby>>*/
-
+        Person p6 = new Person("Evelina", 21);
         HashMap<String, List<String>> hashMap =new HashMap<>();
-        hashMap.put("Silvia", Arrays.asList("dancing", "boxing"));
 
-            HashMap<Person, List<Hobby>> hashMap =new HashMap<>()
-            hashMap.put("Silvia", Arrays.asList("dancing", "boxing"));
+        hashMap.put(String.valueOf(p6), Arrays.asList("dancing", "boxing"));
```

Comment:
this map implementation will use the string version of a Person object instead of the actual object, which will make the map very hard to read and iterate. the commented version with Hashmap<Person, List<Hobby>> is the correct one